Bug 1926054: Update local storage docs about storage class name (#29315)
openshift/openshift-docs: modules/persistent-storage-local-create-cr.adoc

[id="local-volume-cr_{context}"]
= Provisioning the local volumes

> Local volumes cannot be created by dynamic provisioning. Instead, persistent volumes must be created by the Local Storage Operator. This provisioner will look for any devices, both file system and block volumes, at the paths specified in defined resource.

Reviewer: ```suggestion
Local volumes cannot be created by dynamic provisioning. Instead, persistent volumes must be created by the Local Storage Operator. This provisioner looks for any file system or block volumes devices at the paths specified in defined resource.
```

.Procedure

> . Create the local volume resource. This must define the nodes and paths to the local volumes.

Reviewer: ```suggestion
. Create the local volume resource. This resource must define the nodes and paths to the local volumes.
```

          - ip-10-0-158-139
          - ip-10-0-164-33
  storageClassDevices:
    - storageClassName: "local-sc" <3>
      volumeMode: Filesystem <4>
      fsType: xfs <5>
      devicePaths: <6>
        - /path/to/device <7>
----
<1> The namespace where the Local Storage Operator is installed.
<2> Optional: A node selector containing a list of nodes where the local storage volumes are attached. This example uses the node host names, obtained from `oc get node`. If a value is not defined, then the Local Storage Operator will attempt to find matching disks on all available nodes.

> <3> The name of the storage class to use when creating persistent volumes (PV) objects. The Local Storage Operator automatically creates the storage class if it does not exist. Be sure to use a storage class that uniquely identifies this set of local volumes.

Reviewer: ```suggestion
<3> The name of the storage class to use when creating persistent volume objects. The Local Storage Operator automatically creates the storage class if it does not exist. Be sure to use a storage class that uniquely identifies this set of local volumes.
```

          - ip-10-0-140-255
          - ip-10-0-144-180
  storageClassDevices:
    - storageClassName: "localblock-sc" <3>
      volumeMode: Block  <4>
      devicePaths: <5>
        - /path/to/device <6>
----
<1> The namespace where the Local Storage Operator is installed.
<2> Optional: A node selector containing a list of nodes where the local storage volumes are attached. This example uses the node host names, obtained from `oc get node`. If a value is not defined, then the Local Storage Operator will attempt to find matching disks on all available nodes.

> <3> The name of the storage class to use when creating persistent volumes (PV) objects.

Reviewer: ```suggestion
<3> The name of the storage class to use when creating persistent volume objects.
```

<4> The volume mode, either `Filesystem` or `Block`, that defines the type of local volumes.
<5> The path containing a list of local storage devices to choose from.
<6> Replace this value with your actual local disks filepath to the `LocalVolume` resource, such as `/dev/xvdg`. PVs are created for these local disks when the provisioner is deployed successfully.

> . Create the local volume resource in your {product-title} cluster, specifying the file you just created:

Reviewer: ```suggestion
. Create the local volume resource in your {product-title} cluster. Specify the file that you just created:
```
